SU-HSDO/suhumsci
# ARCH-56 default blocks styling

# READY FOR REVIEW

# Summary
- Styles for footer blocks, search, and site title lockup

# Needed By (Date)
- End of sprint (5/24)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test to ensure search is working
3. That footer blocks and site title are working and look good

# Affected Projects or Products
- Archaeology
- suhumsci

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-56

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
